test/e2e/e2e.go

```diff
@@ -1977,7 +1978,64 @@ var _ = ginkgo.Describe("e2e ingress traffic validation", func() {
 				}
 			}
 		})
+		// This test validates ingress traffic to nodeports with externalTrafficPolicy Set to local.
+		// It creates a nodeport service on both udp and tcp, and creates a backend pod on each node.
+		// The backend pod is using the agnhost - netexec command which replies to commands
+		// with different protocols. We use the "hostname" and "clientip" commands to have each backend
+		// pod to reply with its hostname and the request packet's srcIP.
+		// We use an external container to poke the service exposed on the node and ensure that only the
+		// nodeport on the node with the backend actually receives traffic and that the packet is not
+		// SNATed.
+		// In case of dual stack enabled cluster, we iterate over all the nodes ips and try to hit the
+		// endpoints from both each node's ips.
+		ginkgo.It("Should be allowed to node local cluster-networked endpoints by nodeport services with externalTrafficPolicy=local", func() {
```

Comment:
describe where the traffic is coming from. External network container? host? ovn network pod?

Reply:
This is part of the `Validating Ingress Traffic` Context https://github.com/ovn-org/ovn-kubernetes/pull/2136/commits/541aac2900506bbddb457c82223d957030eef673#diff-65b73a6a62a4e5b8903b9f17f0e03a42365b127e7d86f006f9bec363b4be04b8R1941 Which I think describes where the traffic is coming from (External Network container)